packages/tracing/test/browser/request.test.ts

```diff
@@ -47,3 +68,95 @@ describe('registerRequestInstrumentation', () => {
     expect(mockXHRCallback()).toBe(undefined);
   });
 });
+
+describe('fetchCallback', () => {
+  let hub: Hub;
+  let transaction: Transaction;
+  beforeEach(() => {
+    hub = new Hub(new BrowserClient({ tracesSampleRate: 1 }));
+    makeMain(hub);
+    transaction = hub.startTransaction({ name: 'organizations/users/:userid', op: 'pageload' }) as Transaction;
+    hub.configureScope(scope => scope.setSpan(transaction));
+  });
+
+  afterEach(() => {
+    if (transaction) {
+      transaction.finish();
+    }
+    hub.configureScope(scope => scope.setSpan(undefined));
```

Comment:
We create a new hub for each test, so it's unnecessary I guess

Reply:
You are right, I'll change.